### PR TITLE
Make languagePolicy configurable in TestScript

### PR DESCRIPTION
**Pull request by Danieladu**
Original issue: https://github.com/microsoft/botbuilder-dotnet/issues/4143
First PR: https://github.com/microsoft/botbuilder-dotnet/pull/4784

## Description
The first PR make language policy in MultiLanguageRecognizer align with the policy in UseLanguagePolicy,
this PR make the LanguagePolicy configurable in TestScript

## Specific Changes
- Add `LanguagePolicy` property in TestScript
- Remove the parameter assignment of languagePolicy in TestScript

### CODE REVIEW

File: tests/tests.schema [hunk 1]
```diff
@@ -665,7 +665,239 @@
           "description": "Schema to fill in.",
           "anyOf": [
             {
-              "$ref": "#/definitions/schema"
+              "$schema": "http://json-schema.org/draft-07/schema#",
+              "title": "Core schema meta-schema",
+              "definitions": {
+                "schemaArray": {
+                  "type": "array",
+                  "minItems": 1,
```
Comment (chrimc62) [inline tests/tests.schema:673]: If you are going to regenerate this file you need to make sure you are running the latest bf dialog:merge by doing bf plugins:install @microsoft/bf-dialog@next

**Review by chrimc62 — CHANGES_REQUESTED** [review]
:clock1:

**Review by chrimc62 — APPROVED** [review]
:shipit:

**Comment by Danieladu** [comment]
@microsoft/bb-dotnet could you take a look?

**Review by mrivera-ms — APPROVED** [review]

**Pull request merged**